src/Components/NewForm.vue

```diff
@@ -35,6 +35,31 @@
 			rows="4"
 			:placeholder="t('announcementcenter', 'Write announcement text, Markdown can be used …')" />
 
+		<div class="announcement__form__schedule">
+			<NcCheckboxRadioSwitch :checked.sync="scheduleEnabled">
+				{{ t('announcementcenter', 'Schedule announcement time (optional)') }}
+			</NcCheckboxRadioSwitch>
+			<NcDateTimePicker v-model="scheduleTime"
+				:disabled="!scheduleEnabled"
```

Comment:
I think clicking this should automatically tick the checkbox, or the picker should be hidden until checked.

Reply:
Hmm, can we do this in the follow up with the new DatePicker?

Reply:
Yeah see https://github.com/nextcloud/announcementcenter/pull/767#issuecomment-2249611348